README.md

```diff
@@ -169,11 +169,15 @@ c.use Circuitbox::FaradayMiddleware, exceptions: [Faraday::Error::TimeoutError]
 * `default_value` value to return for open circuits, defaults to 503 response
   wrapping the original response given by the service and stored as
   `original_response` property of the returned 503, this can be overwritten
-  either with a static value or a `lambda` which is passed the
-  original_response.
+  with either
+  * a static value
+  * a `lambda` which is passed the `original_response` and `original_error`.
+    `original_response` will be populated if Faraday returne an error response,
+    `original_error` will be populated if an error was thrown before Faraday
+    returned a response.
```

Comment:
Could you add a note that the default_value lambda with arity 1 is deprecated and will be remove in the next version, and also add an issue for that for futur so this does not hang around forever? That would be awesome :)